README.md

```diff
@@ -2,6 +2,49 @@
 ETHOnline 2022 entry by ChainHackers
 https://gamejutsu.app/
 
+# Run locally
+```shell
+yarn install
+```
+add the file ```.env.local``` to the root of the project and in it declare the variable ```NEXT_PUBLIC_ALCHEMY_API_KEY with the key from ALCHEMY_API```
```

Comment:
This needs a more explicit example - i.e. "put a line like the following in your `.env.local`
```properties
NEXT_PUBLIC_ALCHEMY_API_KEY=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```
Most often keys are same-length strings, Alchemy keys are no different. This way it's more obvious.

For single-line formatting, single backticks suffice.

Reply:
corrected